### PR TITLE
Checks: add iOS checkerstyle

### PR DESCRIPTION
**Pull request by dwaynebailey**
Detects $(var) and %@, @1$@ variables

### CODE REVIEW

File: translate/filters/test_checks.py [hunk 1]
```diff
@@ -1097,6 +1098,17 @@ def test_variables_cclicense():
     assert fails_serious(checker.variables, "A @ccvar@", "'n @ccvertaaldeveranderlike@")
 
 
+def test_variables_ios():
+    """Test variables in iOS translations"""
+    # L20n variables
+    ioschecker = checks.IOSChecker()
+    assert passes(ioschecker.variables, "Welcome %@", "Welkom %@")
+    assert fails_serious(ioschecker.variables, "Welcome %@", "Welkom $@")
```
Comment (unho) [inline translate/filters/test_checks.py]: No check for missing `%@` or `$(HELLO)` ? Actually I see no test for `$(HELLO)`

File: translate/filters/checks.py [hunk 2]
```diff
@@ -2394,6 +2394,24 @@ def ftl_format(self, str1, str2):
...
+
+
+class IOSChecker(StandardChecker):
+
+    def __init__(self, **kwargs):
+        checkerconfig = kwargs.get("checkerconfig", None)
+
+        if checkerconfig is None:
+            checkerconfig = CheckerConfig()
+            kwargs["checkerconfig"] = checkerconfig
```
Comment (unho) [inline translate/filters/checks.py]: I am not going to complain about this being copied again and again on every checker class.

File: translate/filters/test_checks.py [hunk 3]
```diff
@@ -1097,6 +1098,17 @@ def test_variables_cclicense():
     assert fails_serious(checker.variables, "A @ccvar@", "'n @ccvertaaldeveranderlike@")
 
 
+def test_variables_ios():
+    """Test variables in iOS translations"""
+    # L20n variables
```
Comment (unho) [inline translate/filters/test_checks.py]: L20n?

File: translate/filters/checks.py (hunk 2 above)
Comment (dwaynebailey) [inline translate/filters/checks.py]: You are... but you aren't, as this is a feature not a refactor ;)

File: translate/filters/test_checks.py (hunk 1 above)
Comment (dwaynebailey) [inline translate/filters/test_checks.py]: Fixed

File: translate/filters/test_checks.py (hunk 3 above)
Comment (dwaynebailey) [inline translate/filters/test_checks.py]: Fixed

**Comment by unho** [comment]
gtm once Travis passes. Very nice work!

**Pull request merged**